light-zk.js/src/wallet/user.ts

```diff
@@ -453,8 +450,8 @@ export class User {
       appUtxo,
       verifierIdl: verifierIdl ? verifierIdl : this.verifierIdl,
       outUtxos,
-      addInUtxos: recipient ? false : true,
-      addOutUtxos: recipient ? false : true,
+      addInUtxos: !recipient,
+      addOutUtxos: !recipient,
```

Comment:
what does this syntax do?

Reply:
It's just a simplified version of previous expression, not(<bool_expression>)